tests/cases/user_test.py

```diff
@@ -549,17 +561,9 @@ def testTemporaryPassword(self):
         }, user=user)
         self.assertStatusOk(resp)
 
-        # Generate an email with a forwarded header
```

Comment:
The purpose of creating the email below was to ensure the behavior of the `X-Forwarded-Host` header that was removed as part of this topic branch. That code is obsolete now that we have the email host setting. This is part of the fix for #955